bundles/org.openhab.binding.paradoxalarm/src/main/resources/OH-INF/thing/partition.xml

```diff
@@ -15,6 +15,7 @@
 		<channels>
 			<channel id="label" typeId="partitionLabel"/>
 			<channel id="state" typeId="state"/>
+			<channel id="detailedState" typeId="detailedState"/>
```

Comment:
Please update the i18n file named paradoxalarm.properties. 2 entries to be added, one for label and one for description.

Please also provide an upgrade instuction XML file so that your new channel can be added automatically to existing things created with UI.

Reply:
> Please also provide an upgrade instuction XML file so that your new channel can be added automatically to existing things created with UI.

Hm.... I'm not sure how it's done. Can you point me to where to see it? I never use the UI. Use the text files only...

Reply:
Disregard the last one. I think I figured out what I need to do (got advised from another PR). Working on it...

Reply:
OK I think I did the requested... Please review the XML as it's the first time I do it. 
There is a thing I don't understand in the documentation... if I change anything in the same channel in a subsequent PR, does that mean that i need to keep version 1 <add channel> and I need to have version 2 with <update channel> and keep a full track for every channel type which gets changed? basically for any change for a specific channel ID I need to have 1, 2, 3, etc... in the XML. Is my understanding correct?

Reply:
Your thing type description (XML) will contain only the last version of your thing type with the higher `thingTypeVersion` property.
But you will keep all the upgrade instructions, allowing the different upgrades even for a user having an old version of the thing type.